laa_court_data_api_app/models/laa_references/external/request/laa_references_patch.py

```diff
@@ -8,13 +8,15 @@ class LaaReferencesPatch(BaseModel):
     defendant_id: UUID | None
     maat_reference: int | None
     unlink_reason_code: int | None
+    unlink_other_reason_text: str | None
 
     class Config:
         schema_extra = {
             "example": {
                 "user_name": "jon-5",
                 "defendant_id": "d7f509e8-309c-4262-a41d-ebbb44deab9e",
-                "maat_reference": 1234567,
-                "unlink_reason_code": 0
+                "maat_reference": 3141592,
+                "unlink_reason_code": 1,
```

Comment:
since there's text in `unlink_other_reason_text` i think the `unlink_reason_code` should be 7.

As seen in [CDUI](https://github.com/ministryofjustice/laa-court-data-ui/blob/master/db/seeds/unlink_reasons.csv)